CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 0.2 (02 November 2016)
+
+* Added Initial Schedules Support (#48)
+* Added Initial Create Group endpoint (#69)
+* Added Connection Credentials for publishing datasources (#80)
+* Added Pager object for handling pagination results and sample (#72, #90)
+* Added ServerInfo endpoint (#84)
+* Code Cleanup
```

Comment:
Changelogs are mostly for end users.  Calling out that we cleaned up code is nice, but the details of that is less important to end users I think.